go-controller/pkg/ovn/service.go

```diff
@@ -168,7 +168,7 @@ func (ovn *Controller) syncServices(services []interface{}) {
 					for lb, hasEps := range svcCacheEntry {
 						if hasEps {
 							klog.Infof("Service Sync: Removing OVN stale reject ACL: %s", name)
-							ovn.removeACLFromNodeSwitches(lb, uuid)
```

Comment:
you need to add code into syncServices to handle removing reject acls from switches then let the add handler  add them back as port group for upgrade.

Reply:
please see my comment above regarding adding it to syncServices.

Reply:
yeah thats fine. I think we still need ovn.removeACLFromNodeSwitches(lb, uuid) here. In case we upgrade from a version that was using ACL on node switches and the ACLs have gone stale.

Reply:
yeah makes sense. Fixed it.

Reply:
@vishnoianil still missing this?

Reply:
Done.